lib/widgets/relax_phrases_generator/phrase_generator.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import 'fade_in.dart';
+
+class PhraseAutoGenerator extends StatefulWidget {
+  @override
+  _PhraseAutoGeneratorState createState() => _PhraseAutoGeneratorState();
+}
+
+class _PhraseAutoGeneratorState extends State<PhraseAutoGenerator> {
+  int _phraseIndex = 0;
+  late List<String> _phraseList;
+
+  @override
+  void didChangeDependencies() {
+    _fillPhraseList();
+    _phraseList.shuffle();
+
+    super.didChangeDependencies();
+  }
+
+  void _fillPhraseList() {
```

Comment:
this method could be omitted and could be moved as part of state class, because you shuffle it in another method, so you dont need to create instances everytime

Reply:
hello, I understand what you mean but I don’t fully understand where it should be, as I see it now in the project I don’t have a state class, do I need to create this class for this one method ?